support-frontend/assets/helpers/formValidation.js

```diff
@@ -44,20 +44,23 @@ export const checkGiftStartDate: (string | null) => boolean = (rawDate) => {
   return isNotEmpty(rawDate) && isNotTooFarInTheFuture(date);
 };
 
-export const checkAmount: (string, CountryGroupId, ContributionType) =>
-  boolean = (input: string, countryGroupId: CountryGroupId, contributionType: ContributionType) =>
-    isNotEmpty(input)
+export const amountIsValid = (
+  input: string,
+  countryGroupId: CountryGroupId,
+  contributionType: ContributionType,
+): boolean =>
+  isNotEmpty(input)
     && isLargerOrEqual(config[countryGroupId][contributionType].min, input)
     && isSmallerOrEqual(config[countryGroupId][contributionType].max, input)
     && maxTwoDecimals(input);
 
 
-export const checkAmountOrOtherAmount: (SelectedAmounts, OtherAmounts, ContributionType, CountryGroupId) => boolean = (
+export const amountOrOtherAmountIsValid = (
```

Comment:
It would be good to refactor this at some point. I think all we actually need to do is check if the amount the user is about to contribute is valid.

Reply:
agreed, it's messy